### DISCUSSION TITLE
thanx for micro g

### DISCUSSION THREAD

**Discussion by mukhsin07** [community/community#70819] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Maaf saya menggunakan bahasa Indonesia,hanya untuk menyampaikan bahwa terimakasih atas aplikasi microg yang telah anda buat ,sungguh sangat simple dan praktis dengan ukuran yang kecil , terimakasih
 Salam dari Indonesia 🇮🇩

**Comment by ghostinhershell** [comment]
Hi @mukhsin07, 

If you're referring to the [microG project](https://github.com/microg) then I would suggest reaching out to the developers for that project and thanking them there. They may not see this message here in our Community. The developers associated with this project should be listed under 'People' in their repo.

Thanks!
